app/scripts/emissions-factors-sync.ts

```diff
@@ -0,0 +1,308 @@
+  await db.models.EmissionsFactor.update(
+    { deprecated: true },
+    { where: {} }
+  );
+}
+
+async function syncPublishers(publishers: PublisherCreationAttributes[]) {
+  logger.debug("Syncing publishers...");
+  for (const publisher of publishers) {
+    await db.models.Publisher.upsert(publisher);
+  }
+  console.log(`Synced ${publishers.length} publishers`);
+}
+
+async function syncDataSources(dataSources: DataSourceI18nCreationAttributes[]) {
+  logger.debug("Syncing data sources...");
+  for (const dataSource of dataSources) {
+    await db.models.DataSource.upsert(dataSource);
+  }
+  console.log(`Synced ${dataSources.length} data sources`);
+}
+
+async function syncMethodologies(methodologies: MethodologyCreationAttributes[]) {
+  logger.debug("Syncing methodologies...");
+  for (const methodology of methodologies) {
+    await db.models.Methodology.upsert(methodology);
+  }
+  console.log(`Synced ${methodologies.length} methodologies`);
+}
+
+async function syncEmissionsFactorsData(emissionsFactors: EmissionsFactorCreationAttributes[]) {
+  logger.debug("Syncing emissions factors...");
+  for (const emissionsFactor of emissionsFactors) {
+    await db.models.EmissionsFactor.upsert(emissionsFactor);
+  }
+  console.log(`Synced ${emissionsFactors.length} emissions factors`);
+}
+
+async function syncDataSourceEmissionsFactorRelationships(relationships: DataSourceEmissionsFactorCreationAttributes[]) {
+  logger.debug("Syncing datasource-emissionsfactor relationships...");
+  await db.models.DataSourceEmissionsFactor.bulkCreate(relationships, {
+    ignoreDuplicates: true
+  });
+  console.log(`Synced ${relationships.length} datasource-emissionsfactor relationships`);
+}
+
+async function deleteUnusedDeprecatedEmissionsFactors() {
+  logger.debug("Deleting unused deprecated emissions factors...");
+  await db.sequelize!.query(`
+    DELETE FROM "EmissionsFactor" ef
+    WHERE ef.deprecated = true AND ef.inventory_id IS NULL AND NOT EXISTS (
+      SELECT FROM "GasValue" gv WHERE gv.emissions_factor_id = ef.id
+    );
+  `);
+  console.log("Deleted unused deprecated emissions factors");
+}
+
+async function syncEmissionsFactors() {
+  const projectDir = process.cwd();
+  env.loadEnvConfig(projectDir);
+  const SKIP_TIMESTAMP_CHECK = process.env.SKIP_TIMESTAMP_CHECK === "true";
+
+  const GLOBAL_API_URL =
+    process.env.GLOBAL_API_URL || "http://api.citycatalyst.io";
+  console.log("Using global API at", GLOBAL_API_URL);
+
+  if (!db.initialized) {
+    await db.initialize();
+  }
+
+  let catalogue = await db.models.Catalogue.findOne({
+    where: { type: "emissions_factors" },
+  });
+  if (!catalogue) {
+    catalogue = await db.models.Catalogue.create({
+      type: "emissions_factors",
+      lastUpdate: new Date(0),
+    });
+  }
+
+  const catalogueUrl = `${GLOBAL_API_URL}/api/v0`;
+
+  if (!SKIP_TIMESTAMP_CHECK) {
+    console.warn("Timestamp checking not implemented for emissions factors yet, proceeding with sync");
+  }
```

Comment:
Can you create a tech debt ticket for this and assign it to one of the following sprints? Thanks 👍